bower.json

```diff
@@ -22,27 +22,12 @@
     "lib"
   ],
   "dependencies": {
-    "bootstrap-sass-official": "3.3.1",
-    "DataTables": "1.10.13",
-    "es5-shim": "2.3.0",
     "excanvas": "*",
```

Comment:
We can fork this one and add a package.json. We wont need this in the future of WET so we're just keeping it to not breaking existing things.

Reply:
Yup, but that can be done then updated in a separate PR

Reply:
I agree!